### ISSUE TITLE
KIF crashes the app when a test fails

### ISSUE THREAD

**Issue by WenxingYan** [kif-framework/KIF#746]
Anyone has similar issues? Rather than just failing the test, it's throwing a EXC_BAD_ACCESS and crashes the app when a test is failing(for example, failed to find accessibility element)

**Comment by mikelupo** [comment]
Are you testing in a web UI? I've seen something like this on an occasion. Oh, also, what version of KIF are you using?

**Comment by WenxingYan** [comment]
I'm using KIF 3.3. It's crashing my app for nearly every failing test, only when running from a script, but not related with any web UI testing.

**Comment by mikelupo** [comment]
Are you integrated via cocoapods? When did this start happening?
